src/include/migraphx/tensor_view.hpp

```diff
@@ -119,6 +119,18 @@ struct tensor_view
         return m_data[m_shape.index(i)];
     }
 
+    template <class Range>
```

Comment:
This should be `template <class Range, MIGRAPHX_REQUIRES(not std::is_integer<Range>{})>` so other integer indices types(besides `std::size_t`) wont use this overload.

Reply:
I have `decltype()` which would fail if it is integer.